zebra-consensus/src/transaction/check.rs

```diff
@@ -330,6 +330,10 @@ pub fn coinbase_outputs_are_decryptable(
     network: &Network,
     height: Height,
 ) -> Result<(), TransactionError> {
+    if !transaction.is_coinbase() {
+        return Ok(());
+    }
+
```

Comment:
Unfortunately this seems to cause a test failure, https://github.com/ZcashFoundation/zebra/actions/runs/12747335909/job/35525212184?pr=9098#step:12:2959

```suggestion
```

Reply:
When looking into this, I found a simple optimization that I think is worth doing, and we could also keep this check. I'll open another PR.